marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/InputEndpointImpl.java

```diff
@@ -83,12 +92,53 @@ private int getBatchSize() {
 
 		@Override
 		public void accept(InputStream input) {
-			// TODO Auto-generated method stub
+			
+			InputStream output = null;
+			try {
+				queue.put(input);
+			} catch (InterruptedException e) {
+				e.printStackTrace();
```

Comment:
If the bulk caller can't accept the input, is that a condition that the caller would need to know about?

Would it make sense to rethrow the exception as a wrapped runtime exception?